src/shipit_static_analysis/shipit_static_analysis/report/base.py

```diff
@@ -61,25 +63,48 @@ def requires(self, configuration, *keys):
 
         return out
 
+    def calc_stats(self, issues):
+        '''
+        Calc stats about issues:
```

Comment:
Nit: What are "calc stats"?

Reply:
It's a function that calculate basic statistics about issues, grouped by their analyzer.
This is used to display the number of publishable issues in reporting tools & email